docs/index.d.ts

```diff
@@ -200,6 +202,8 @@ export interface CoreNodePoxResponse {
   rejection_fraction: number;
   reward_cycle_id: number;
   reward_cycle_length: number;
+  rejection_votes_left_required: number;
+  total_liquid_supply_ustx: number;
```

Comment:
The field seems to be inconsistent with the other fields using a string to represent large integer values (e.g. `balance`, `locked`). Not going to hold up this PR over it though.

Reply:
cc @agraebe